backend/server.js

```diff
@@ -50,6 +52,33 @@ app.get('/config', (req, res) => {
   });
 });
 
+/**
+ * Webhook example
+ *
+ * To test this, use ngrok to make the app publically available, then add the URL to your webhook settings here: https://dashboard.stripe.com/account/webhooks
+ * Make sure you added the webhook secret to your .env file, then click the "Send test webhook" button in your webhook configuration page
+ */
+const webhookSecret = process.env.STRIPE_WEBHOOK_SECRET;
+
+app.post('/webhooks', bodyParser.raw({type: 'application/json'}), (req, res) => {
+  const sig = req.headers['stripe-signature'];
+
+  let event;
+
+  try {
+    event = stripe.webhooks.constructEvent(req.body, sig, webhookSecret);
+  } catch  (err) {
+     // On error, return the error message
+     return res.status(500).send(`Webhook Error: ${err.message}`);
```

Comment:
Shouldn't this send a 400? It's not technically a server error, no? It's also not really a bad request, but that's what our examples use today: https://stripe.com/docs/webhooks/signatures#verify-official-libraries
Also, we probably should update the Node example in the docs[0] with your neater webhook signature handling example. Thanks for that!

[0] https://stripe.com/docs/webhooks/signatures#verify-official-libraries